### DISCUSSION TITLE
Homeworker.li Icon

### DISCUSSION THREAD

**Discussion by matheine** [elrumo/macOS_Big_Sur_icons_replacements#1136] [Icon request]
Can anyone make me a futuristic icon in macOS Big Sur style from Homeworker?
![Logo](https://user-images.githubusercontent.com/68849310/113573640-cf875100-961a-11eb-8f31-7850ae02d902.png)

**Comment by cake03** [comment]
## Download : https://user-images.githubusercontent.com/80169337/113662611-0e261580-96c6-11eb-9dd6-45d997aab9a5.png
### Preview Image: ![li Preview image](https://user-images.githubusercontent.com/80169337/113662611-0e261580-96c6-11eb-9dd6-45d997aab9a5.png)

**Reply by matheine** [reply]
Thank You 🙏

**Reply by matheine** [reply]
How do you make those icons? Which app?

**Reply by cake03** [reply]
I've Made it with Sketch

**Reply by matheine** [reply]
Okay thanks
